pkg/apis/servicecatalog/fake/pointers.go

```diff
@@ -0,0 +1,5 @@
+package fake
+
+func boolPtr(b bool) *bool {
+	return &b
```

Comment:
various fields need to store `*bool`s, and this is a convenience function to create them with a one-liner